cfgov/paying_for_college/models/blocks.py

```diff
@@ -0,0 +1,57 @@
+from wagtail.wagtailcore import blocks
+
+
+class QuizAnswers(blocks.StructBlock):
+    answers = blocks.ListBlock(
+        blocks.StructBlock([
+            ('choice_letter', blocks.CharBlock(
+                max_length=20, required=False)),
+            ('answer_choice', blocks.CharBlock(max_length=500)),
+            ('answer_response', blocks.RichTextBlock(
+                features=[
+                    'ol', 'ul', 'bold', 'italic',
+                    'link', 'image', 'document-link',
+                ],
+                blank=True,
+                required=False,
+            )),
+        ]))
+
+    class Meta:
+        template = 'paying-for-college/quiz-answers.html'
+
+
+class GuidedQuiz(blocks.StructBlock):
+    situation = blocks.RichTextBlock(
+        features=[
+            'ol', 'ul', 'bold', 'italic', 'link', 'image', 'document-link',
+        ],
+        blank=True,
+        required=False,
+    )
+    questions = blocks.ListBlock(
+        blocks.StructBlock([
```

Comment:
Is there supposed to be a field to indicate what the correct answer for a question is?

Reply:
The "quiz" is less a test or competition than it is an attempt to educate in an engaging way, so we don't have a right or wrong flag, and we're not keeping score. 
The "reveal" of right or wrong would come in the explanations for each question.

Reply:
The quiz block might be more likely to be reused if it had more gamelike elements, but it's not the intent here, and the YES project was looking at this same strategy and might pick it up.